### PR TITLE
Add imagePullSecrets option to ServiceAccount in helm chart

### PR DESCRIPTION
**Pull request by BrianKopp**
### Description

This PR adds the ability to configure the ServiceAccount to have imagePullSecrets for docker secrets to authenticate against docker registries. This is for the aws-load-balancer-controller chart.

### Checklist
- [ ] Added tests that cover your change (if possible)
- [x] Added/modified documentation as required (such as the `README.md`, or the `docs` directory)
- [x] Manually tested
- [x] Made sure the title of the PR is a good description that can go into the release notes

### BONUS POINTS checklist: complete for good vibes and maybe prizes?! :exploding_head:
- [ ] Backfilled missing tests for code in same general area :tada:
- [ ] Refactored something and made the world a better place :star2:

### CODE REVIEW

File: helm/aws-load-balancer-controller/values.yaml [hunk 1]
```diff
@@ -26,6 +26,12 @@ serviceAccount:
   name:
   # Automount API credentials for a Service Account.
   automountServiceAccountToken: true
+  # List of image pull secrets to add to the Service Account.
+  # e.g.
+  # imagePullSecrets
+  # - name: docker
+  imagePullSecrets:
+  - name: docker
```
Comment (kishorj) [inline helm/aws-load-balancer-controller/values.yaml:34]: default should be empty

File: helm/aws-load-balancer-controller/values.yaml [hunk 2]
```diff
@@ -26,6 +26,12 @@ serviceAccount:
   name:
   # Automount API credentials for a Service Account.
   automountServiceAccountToken: true
+  # List of image pull secrets to add to the Service Account.
+  # e.g.
+  # imagePullSecrets
+  # - name: docker
+  imagePullSecrets:
```
Comment (kishorj) [inline helm/aws-load-balancer-controller/values.yaml:33]: How about this for the comment and the value:
```
# imagePullSecrets specifies the list of image pull secrets to add to the ServiceAccount
imagePullSecrets:
    # - name: docker
```

File: helm/aws-load-balancer-controller/values.yaml (hunk 1 above)
Comment (BrianKopp) [reply helm/aws-load-balancer-controller/values.yaml]: Shoot, I left this on for testing that the values piped through correctly. I've updated the values and comments as you've suggested.

File: helm/aws-load-balancer-controller/values.yaml (hunk 2 above)
Comment (BrianKopp) [reply helm/aws-load-balancer-controller/values.yaml:30]: Updated!

File: helm/aws-load-balancer-controller/values.yaml [hunk 3]
```diff
@@ -26,6 +26,9 @@ serviceAccount:
   name:
   # Automount API credentials for a Service Account.
   automountServiceAccountToken: true
+  # List of image pull secrets to add to the Service Account.
```
Comment (kishorj) [inline helm/aws-load-balancer-controller/values.yaml:29]:
```suggestion
  # imagePullSecrets specifies the list of image pull secrets to add to the Service Account.
```

File: helm/aws-load-balancer-controller/values.yaml [hunk 4]
```diff
@@ -26,6 +26,9 @@ serviceAccount:
   name:
   # Automount API credentials for a Service Account.
   automountServiceAccountToken: true
+  # imagePullSecrets specifies the list of image pull secrets to add to the Service Account.
```
Comment (kishorj) [inline helm/aws-load-balancer-controller/values.yaml:29]:
```suggestion
  # List of image pull secrets to add to the Service Account.
```

File: helm/aws-load-balancer-controller/values.yaml (hunk 2 above)
Comment (kishorj) [reply helm/aws-load-balancer-controller/values.yaml:30]: Thanks

**Review by kishorj — APPROVED** [review]

**Pull request merged**